### PR TITLE
Update Webstorm integration docs

### PR DESCRIPTION
**Pull request by alzuma**
This is where I found File Watchers (File -> Settings). Couldn't find Preferences anywhere. Working with Windows 10. WS 2018.3

- [] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory)
- [x] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/master/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

**Comment by SimenB** [comment]
It should be OS agnostic, not just convert from Mac to Windows

**Comment by alzuma** [comment]
@SimenB  Any suggestions on how to make it OS agnostic? As right now it is not straightforward how to add File Watchers :)

File: website/versioned_docs/version-stable/webstorm.md [hunk 1]
```diff
@@ -18,9 +18,9 @@ For older IDE versions, please follow the instructions below.
 
 To automatically format your files using `prettier` on save, you can use a [File Watcher](https://plugins.jetbrains.com/plugin/7177-file-watchers).
 
-Go to _Preferences | Tools | File Watchers_ and click **+** to add a new watcher.
+Go to _File | Settings | Tools | File Watchers_ and click **+** to add a new watcher.
```
Comment (kachkaev) [inline website/versioned_docs/version-stable/webstorm.md]: Is this an actual update in the app’s menu or just a difference between macOS / Windows?

**Comment by alzuma** [comment]
@SimenB maybe with keyboard shortcuts to open the config window. For Windows it is Ctrl+Alt+S

File: website/versioned_docs/version-stable/webstorm.md (hunk 1 above)
Comment (alzuma) [reply website/versioned_docs/version-stable/webstorm.md]: I don't know, don't have Mac :/

**Comment by SimenB** [comment]
Mac is `⌘ + ,`, which is standard for that action across pretty much all applications. So that could work instead of finding it in the menus.

---

JetBrains itself allows you to switch the keymap in their docs (see upper right in https://www.jetbrains.com/help/idea/using-file-watchers.html), so not much help there.

**Comment by alzuma** [comment]
Cool, cool will make changes in the patch then.

**Comment by ikatyang** [comment]
cc @prigara

File: website/versioned_docs/version-stable/webstorm.md [hunk 2]
```diff
@@ -18,9 +18,9 @@ For older IDE versions, please follow the instructions below.
 
 To automatically format your files using `prettier` on save, you can use a [File Watcher](https://plugins.jetbrains.com/plugin/7177-file-watchers).
 
-Go to _Preferences | Tools | File Watchers_ and click **+** to add a new watcher.
+In the _Settings/Preferences_ dialog (`⌘,` on Mac or `Ctrl+Alt+S` on Windows), click **File Watchers** under **Tools** and click **+** to add a new watcher.
 
-In Webstorm 2018.2, select Prettier from the list, review the configuration, add any additional arguments if needed, and click OK.
+In Webstorm 2018.3, select Prettier from the list, review the configuration, add any additional arguments if needed, and click OK.
```
Comment (prigara) [inline website/versioned_docs/version-stable/webstorm.md]: I'd suggest "In WebStorm 2018.2 or above" or remove that bit completely and start with "Select Prettier".

**Comment by ikatyang** [comment]
Thanks!

**Pull request merged**